src/rpc/api.ts

```diff
@@ -129,6 +129,55 @@ export namespace Api {
     createdAt?: number;
   }
 
+  export interface GetTransactionsRequest {
+    startLedger: number;
+    cursor?: string;
+    limit?: number;
+  }
+
+  export interface RawTransactionInfo {
+    status: GetTransactionStatus;
+    ledger: number;
+    createdAt: number;
+    applicationOrder: number;
+    feeBump: boolean;
+    envelopeXdr?: string;
+    resultXdr?: string;
+    resultMetaXdr?: string;
+    diagnosticEventsXdr?: string[];
+  }
+
+  export interface TransactionInfo {
+    status: GetTransactionStatus;
+    ledger: number;
+    createdAt: number;
+    applicationOrder: number;
+    feeBump: boolean;
+    envelopeXdr: xdr.TransactionEnvelope;
+    resultXdr: xdr.TransactionResult;
+    resultMetaXdr: xdr.TransactionMeta;
+    returnValue?: xdr.ScVal;
+    diagnosticEventsXdr?: string[];
```

Comment:
Shouldn't this be an `xdr.DiagnosticEvent[]`?

Reply:
Ah good catch